vutil/vxs.inc

```diff
@@ -322,7 +322,7 @@ VXS(version_vcmp)
 	       SV	*rs;
 	       SV	*rvs;
 	       SV * robj = ST(1);
-	       const IV	 swap = (IV)SvIV(ST(2));
+	       const int swap = items > 2 ? SvTRUE(ST(2)) : 0;
```

Comment:
@haarg should not we customize the version in blead with this fix?
ping @ZenoParadoxus

Reply:
I just took care of the final permissions, I'll put a new release out

Reply:
side topic: I wonder if this should be a `dist` and not a `cpan` module on blead